### PR TITLE
CBF Controller for Double-Integrator dynamics

### PR DESCRIPTION
**Pull request by luigiberducci**
Hi all :-)
I have been implementing a simple training script for neural cbf controllers on double-integrator dynamics.
This was part of a private project but I thought it might be interesting other members of the community.

The training is still on-going, that's why I opened it as draft PR. 
As this is the first use of this repository, any feedback on the implementation is welcome.

### Changes
- Implemented a new system with double-integrator dynamics.
- Created script for training a neural cbf controller.
- To avoid using gurobi, the `disable_gurobi` argument is added to  the CBFController and then forwarded to the parent class CLFController.

### Experiments
I run the script as it is and collected snapshot of the tensorboard after 50 epochs.

Despite the overall trend seems to go in the right direction and the positive and negative regions in the barrier function roughly correspond to safe/unsafe set, I cannot see sign of life in the rollouts. 
Do you have any insights on this?

**Accuracy**
image

**QP-relaxation**
image

**Loss**
image

**Rollouts**
image

**V-contour:**
image

### CODE REVIEW

**Comment by luigiberducci** [comment]
Opened issue #33.
